### PR TITLE
Fixes #10673: Add a "general presentation" chapter in documentation

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10673

### CODE REVIEW

**Comment by amousset** [comment]
These images needs to be translated:

* images/introduction/web_api_cli.png
* images/introduction/ncf_language.png
*  images/introduction/build_run_devops.png

And this only is outdated (3.0 at first sight):

* images/introduction/agent_output.png

**Comment by fanf** [comment]
Yeah, I know. But I don't have the sources, and I think it may be more effecient if the person in charge of each chapter do the screenshoting work when needed. It put them to give a general idea about what could go in each part;

**Comment by amousset** [comment]
I don't really see the difference between "Key Points" and "Key Features" from their name. Maybe "Key Points" should got o the "What is Rudder page", or get another name.

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
As proposed, I merged key points with the intro.

File: 00_introduction/01_presentation.txt [hunk 1]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+==== Made for the Production
+
+We believe that there is a growing impedence mismatch between the Short Time of
+application development and deployement, and the Long Time of the infrastructure.
+The latter need rationalisation, stability and conformity before catching the hyped
+techno of the day, to be able to deliver reliable technical platform, continuously
+working with a minimum of risks.
+
+Rudder was made for the Long Time, to help team deliver efficient infrastructures with
+simplicity, giving them feedback where needed, keeping them alert of possibile
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: possible

File: 00_introduction/01_presentation.txt [hunk 2]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+incoming problem, continously checking conformity to their rules, and all of that
+whatever the infrastructure they choose to build.
+
+image::./images/introduction/build_run_devops.png[Modern IT production of services and Open Source automation tools stack]
+
+To achieve these goals, Rudder go beyond simple automation of commands or
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: goes

File: 00_introduction/01_presentation.txt [hunk 3]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+configurations. Rudder continuously maintains your infrastructure to keep it
+conforms with your configurations and security rules.
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: conform

File: 00_introduction/01_presentation.txt [hunk 4]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+
+At each level (global, by configuration policy, by node, etc), you can choose to
+either *Audit* the component - and no modification at all will made on it -, or to
+*Enforce* the policy, automatically correcting a drift if needed.
+
+==== Different roles for a better accessibility
+
+Rudder was thought from the start for plug&playability: easy to install and to
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: play ability

File: 00_introduction/01_presentation.txt [hunk 5]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+upgrade, easy to start with and growth with.
+
+Rudder comes with a graphical interface, a standard library of configuration
+policy ready to use, and a graphical rule editor.
+
+image::./images/introduction/web_api_cli.png[Use what best feets your need: Web interface, API, or console]
+
+Developers can scripts Rudder throught its APIs and security teams can check
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: script

File: 00_introduction/01_presentation.txt [hunk 6]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+conformity level to their policies or inventory (both software and hardware) of a
+server at any time.
+
+
+==== Universality
+
+Rudder agent is extremely fast, light, and versatil. It works on a wide variety
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: versatile

File: 00_introduction/01_presentation.txt [hunk 7]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+of OS or hardware, from physical server to cloud instance, user laptops or even
+Digital Cities and IoT objects.
+
+image::./images/introduction/agent_output.png[Versatil agent]
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: versatile

**Comment by fanf** [comment]
PR rebased

File: 00_introduction/01_presentation.txt [hunk 8]
```diff
@@ -40,3 +40,48 @@ The biggest known deployment in 2017 is about 7000 nodes.
...
+server at any time.
+
+
+==== Universality
+
+Rudder agent is extremely fast, light, and versatile. It works on a wide variety
+of OS or hardware, from physical server to cloud instance, user laptops or even
+Digital Cities and IoT objects.
+
+image::./images/introduction/agent_output.png[Versatile agent]
```
Comment (amousset) [inline 00_introduction/01_presentation.txt]: Could you insert a newline here?

File: 00_introduction/20_key_features.txt [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+==== Extensibilty
+
+image::./images/introduction/generic_methods_list.png[Non exhaustive list of generic methods]
+
+image::./images/introduction/rule_directive_generic_method_stack.png[Build your own configuration, matching your requirements]
+
+image::./images/introduction/ncf_language.png[high level definition language]
+
+image::./images/introduction/technique_editor_overview.png[Graphical Technique Editor - the simplest way to build new configuration]
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: Could you insert a newline here?

File: 00_introduction/40_architecture_and_dependencies.txt [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+
+image::./images/introduction/rudder_functional_component_diagram-simple-v1.png[Rudder functionnal architecture]
+
+
+==== Network architecture in client/server mode
+
+image::./images/introduction/network_connections.png[Network architecture]
+
+
+==== Agents
```
Comment (amousset) [inline 00_introduction/40_architecture_and_dependencies.txt]: Could you insert a newline here?

**Comment by fanf** [comment]
PR rebased

**Pull request merged**